static/src/javascripts/bootstraps/liveblog.js

```diff
@@ -192,7 +192,8 @@ define([
                 dropdown.addClass('dropdown--active');
                 dropdowns.updateAria(dropdown);
 
-                if (detect.isBreakpoint({ min: 'desktop' }) && config.page.keywordIds.indexOf('football/football') < 0) {
+                if (detect.isBreakpoint({ min: 'desktop' }) && config.page.keywordIds.indexOf('football/football') < 0 && config.page.keywordIds.indexOf('sport/rugby-union') < 0) {
+                    
```

Comment:
@rich-nguyen this uses same tag we added for `content.scala` with checking it is a rugby match https://github.com/guardian/frontend/blob/master/common/app/model/meta.scala#L356-L357
Do you think this is okay?